common/OpTestHMC.py

```diff
@@ -59,19 +59,36 @@ class OpSecureBootUtilities():
     '''
     This class addresses the secureboot related pre-requisites.
     '''
-
-    def __init__(self, console=None, distro=None):
+    def __init__(self, cv_hmc=None, hmc_con=None, console=None, distro=None):
+        self.cv_hmc = cv_hmc
+        self.hmc_con = hmc_con
         self.console = console
         self.distro = distro
 
+    def check_hmc_level_secureboot_state(self):
```

Comment:
I guess reference to HMC is to convey that we are using HMC to check the value of secure boot. In that case how about check_lpar_secureboot_state?

Reply:
Done